src/webchat-ui/components/plugins/input/file/PreviewUploadedFiles.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { styled } from '../../../../style';
+import CloseIcon from "../../../../assets/baseline-close-24px.svg";
+import IconButton from '../../../presentational/IconButton';
+import LinearProgressBar from '../../../presentational/LinearProgressBar';
+
+const UploadedFilesContainer = styled.div(({theme}) => ({
+    padding: `${theme.unitSize}px ${theme.unitSize * 2}px`,
+    display: 'flex',
+    flexDirection: 'row',
+    overflowX: 'auto',
+    overflowY: 'hidden',
+    height: theme.unitSize * 7,
```

Comment:
can you explain when we use unitSizes and when not in the webchat? Asking bc I also see a lot of fixed values